webExtensionApis/study/api.md

```diff
@@ -196,6 +196,42 @@ Usage scenarios:
   * $ref:
   * optional: false
 
+### `browser.study.log( value1, value2, value3, value4, value5 )`
```

Comment:
This needs an issues to discuss.  I dropped it originally because of these issues:

1.  5 arguments feels arbitrary.  
2.  How do you control what level this logging is at?
3.  None of this feels idiomatic.

Counter suggestion:  
-  browser.log.{info,log} whatever api.  

Console.log is how we are supposed to log, but it all feels very unresolved.

Reply:
Let's discuss further in https://github.com/mozilla/shield-studies-addon-utils/issues/139